resources/dicts/patrols/plains/training/any.json

```diff
@@ -2828,5 +2828,125 @@
+                "exp": 10,
+                "stat_trait": ["fierce","bold","confident","competitive","ambitious","rebellious","daring","adventurous","cunning"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l's heart lurches seeing the apprentice so low. Could p_l have done something to make app1 feel this way? Not having any words p_l sits close to app1, wrapping {PRONOUN/p_l/poss} tail around {PRONOUN/app1/poss}. P_l purrs softly, reassuringly. Until app1 purrs back, gratefully. ",
+                "exp": 10,
+                "stat_trait": ["insecure","nervous","oblivious","careful","responsible"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            },
+            {
+                "text": "p_l shifts awkwardly at app1's side until {PRONOUN/p_l/subject} can't stand the silence any longer. P_l drops into a crouch and nudges app1 with {PRONOUN/p_l/poss} paw. App1 looks back, confused as p_l keeps bouncing and poking at {PRONOUN/p_l/poss} with a goofy expression on {PRONOUN/p_l/poss} face, until app1 gives in. The warrior and apprentice play like kits, and the smile on app1's face says {PRONOUN/app1/subject} {VERB/app1/have/has} forgotten all about their doubt.",
+                "exp": 10,
+                "stat_trait": ["charismatic","flamboyant","playful","childish","shameless","strange","sneaky"],
+                "relationships": [
+                    {
+                        "cat_to": ["p_l"],
+                        "cat_from": ["app1"],
+                        "values": ["platonic", "comfort", "respect", "trust"],
+                        "amount": 10
+                    } ],
+                "weight": 20
+            }
+        ],
+        "fail_outcomes": [
+            {
+                "text": "P_l tries to think of something to say or do, anything, but {PRONOUN/p_l/subject} fumble as app1 becomes more and more unconsolable. Finally, p_l gives up. Leaving app1 alone with {PRONOUN/app1/poss} thoughts.",
```

Comment:
lol p_l you aint't a great mentor huh

also needs the "can_have_stat": ["p_l"], and s_c reference in text rather than p_l
can't capitalise p_l